git-helpers/src/remote_helper.rs

```diff
@@ -31,6 +31,14 @@ const SECRET_KEY_FILE: &str = "librad.key";
 pub fn run() -> anyhow::Result<()> {
     let url = {
         let args = env::args().skip(1).take(2).collect::<Vec<_>>();
+        if args.is_empty() {
+            return Err(anyhow::anyhow!(
```

Comment:
You can avoid the escapes in the string by using the [Raw string literals](https://doc.rust-lang.org/reference/tokens.html#raw-string-literals)

Reply:
Thanks for the review ! I did not know which one to choose. I made the change !